pkg/controllers/dashboard/fleetcharts/controller.go

```diff
@@ -119,6 +124,15 @@ func (h *handler) onSetting(key string, setting *v3.Setting) (*v3.Setting, error
 		},
 	}
 
+	if extraValues, err := h.chartsConfig.GetChartValues(chart.FleetChartName); err != nil {
+		// Missing extra config is okay, return the error otherwise
+		if !chart.IsNotFoundError(err) {
+			return nil, err
+		}
+	} else {
+		coalesceMap(fleetChartValues, extraValues)
+	}
+
```

Comment:
I am wondering how much sense it could make to move this further down. In other words, how likely might it be that we'd need to set gitjob values? Probably not much, but if we did, my understanding is that they would be overwritten at L158.

Reply:
Wow, nice catch, thanks!